apps/prairielearn/src/ee/lib/ai-grading.ts

```diff
@@ -108,57 +112,44 @@ async function generateGPTPrompt({
   // Examples
   for (const example of example_submissions) {
     if (rubric_items.length > 0 && example.manual_rubric_grading_id) {
+      // Note that the example may have been graded with a different rubric,
+      // or the rubric may have changed significantly since the example was graded.
+      // We'll show whatever items were selected anyways, since it'll likely
+      // still be useful context to the LLM.
       const rubric_grading_items = await queryRows(
         sql.select_rubric_grading_items,
         {
           manual_rubric_grading_id: example.manual_rubric_grading_id,
         },
         RubricItemSchema,
       );
-      // Warning when graded rubric item does not match current rubric item
-      if (
-        rubric_grading_items.length > 0 &&
-        rubric_grading_items[0].rubric_id !== rubric_items[0].rubric_id
-      ) {
-        warning += `Instance question ${example.instance_question_id}: example rubric id is different from the current rubric id.\n`;
-      }
       let rubric_grading_info = '';
       for (const item of rubric_grading_items) {
         rubric_grading_info += `description: ${item.description}\n`;
       }
       messages.push({
         role: 'user',
-        content: `Example answer: \n<answer>\n${example.submission_text} \n<answer>\nSelected rubric items for this example answer: \n${rubric_grading_info}`,
+        content: `Example response: \n<response>\n${example.submission_text} \n<response>\nSelected rubric items for this example response: \n${rubric_grading_info}`,
```

Comment:
We now use "response" instead of "answer" to refer to what the student submitted (https://github.com/PrairieLearn/PrairieLearn/pull/10726#discussion_r1964661052).

Reply:
I added changes to suggest using "student response" for extra clarity. Please edit as you see fit to maximize clarify. It's possible that we should reserve "student response" for the actual student response needing grading, and "example response" for the RAG. I'm not sure.

Reply:
I like it, I accepted all the suggestions!